xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-test/xwiki-platform-appwithinminutes-test-pageobjects/src/main/java/org/xwiki/appwithinminutes/test/po/StaticListClassFieldEditPane.java

```diff
@@ -47,4 +47,13 @@ public StaticListItemsEditor getItemsEditor()
         return new StaticListItemsEditor(
             getDriver().findElementWithoutWaiting(getContainer(), By.className("staticListEditor")));
     }
+
+    /**
+     * @return {@code true} is the field is readonly, {@code false} otherwise
+     * @since 13.2
+     */
+    public boolean isReadOnly()
```

Comment:
Missing unstable

Reply:
Finally not needed, see https://matrix.to/#/!ikPtGZaGWtyblizzlR:matrix.xwiki.com/$16165824797107jQKPF:matrix.xwiki.com?via=matrix.org&via=matrix.xwiki.com&via=dodsorf.as
But i'm missing a 12.10.6 since annotation, fixing that too.